### ISSUE TITLE
Tool generate wrong sql statement when default parameter is set

### ISSUE THREAD

**Issue by alvlapo** [phalcon/phalcon-devtools#1210]
### Syntax error: 7 ERROR:  syntax error at or near "DEFAULT"

When i add default parameter to my field

```php
        new Column(
          'is_one_address',
          [
              'type' => Column::TYPE_BOOLEAN,
              'default' => false,
              'after' => 'postal_address'
          ]
        ),
```
and run migration, i have error report in console
```php
Phalcon DevTools (3.2.12)

ERROR: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "DEFAULT"
LINE 2:   DEFAULT true"id" SERIAL NOT NULL,
```
and in postgresql log a see

```sql
ERROR:  syntax error at or near "DEFAULT" at character 38
postgres_1     | STATEMENT:  CREATE TABLE "public"."profiles" (
postgres_1     |                 DEFAULT true"id" SERIAL NOT NULL,
postgres_1     |                "inn" CHARACTER VARYING(12),
postgres_1     |                "type" CHARACTER VARYING(16),
postgres_1     |                "alter_title" CHARACTER VARYING(64),
postgres_1     |                "about" TEXT,
postgres_1     |                "actual_address" JSON,
postgres_1     |                "postal_address" JSON,
postgres_1     |                "is_one_address" BOOLEAN,
```

### Details

Environment:
*  OS: Linux 407ad79a6c95 4.13.0-37-generic #42-Ubuntu SMP Wed Mar 7 14:13:23 UTC 2018 x86_64
*  PHP Version: 5.6.11-1ubuntu3.4
*  PHP SAPI: cli
*  PHP Bin: /usr/bin/php5
*  PHP Extension Dir: /usr/lib/php5/20131226
*  PHP Bin Dir: /usr/bin
*  Loaded PHP config: /etc/php5/cli/php.ini
Versions:
*  Phalcon DevTools Version: 3.2.12
*  Phalcon Version: 3.2.2
*  AdminLTE Version: 2.3.6

**Comment by alvlapo** [comment]
Migration file was generated by phalcon devtool

**Comment by sergeysviridenko** [comment]
@alvlapo Could you try phalcon version 3.3? This problem was fixed in 3.3.x branch.
 [PR link](https://github.com/phalcon/cphalcon/pull/13165)

**Comment by sergeyklay** [comment]
I'm closing this issue due to the lack of any reaction. I'll open it again if the need arises
